### ISSUE TITLE
FIFA 99: Players are floating (x64 Dynarec)

### ISSUE THREAD

**Issue by loganmc10** [mupen64plus/mupen64plus-core#286]
Works fine with the Cached Interpreter. When I tried the x86 new dynarec, it crashed, still need to figure out where it crashed.

![fifa_99-003](https://cloud.githubusercontent.com/assets/848146/26377665/7548574e-3fce-11e7-8493-bebd5d92529d.png)
![fifa_99-007](https://cloud.githubusercontent.com/assets/848146/26377700/a08d2f38-3fce-11e7-9968-9695e8e97977.png)

**Comment by loganmc10** [comment]
Here is the assert in the new dynarec @Gillou68310 :

```
mupen64plus: ../../src/device/r4300/new_dynarec/new_dynarec.c:6356: cjump_assemble: Assertion `s1h>=0' failed.
```

**Comment by Gillou68310** [comment]
Thanks for opening the issue I don't have time to look at it right now but @bsmiles32 could you assign this issue to me so I can remember it?

**Comment by loganmc10** [comment]
This problem doesn't occur in the x86 "old" dynarec.

Enableding ```INTERPRET_ADDU``` fixes the problem in the x86 dynarec. I'm not really an assembly code guy, but it looks like the problem is with the ADDU instruction.

**Comment by DaveGravel** [comment]
Here I get this issue without the new dynarec.
If I enable the new dynarec it look to work correctly.
You can see here in this video, I use the 64bits version:
https://youtu.be/_P8pzxo0YFk?t=90

**Comment by Gillou68310** [comment]
@loganmc10 should be fixed by #871

**Comment by loganmc10** [comment]
Yes looks good, I'll close this so I don't forget
